File: lwjgl/src/main/java/com/darkraha/opengldemoj/gl/shader/HelperBuilder.java
package com.darkraha.opengldemoj.gl.shader;

public  interface HelperBuilder{
    void addTypeDeclarations(StringBuilder sb, int shaderType);
    void addInputDeclarations(StringBuilder sb, int shaderType);
    void addExchangeDeclarations(StringBuilder sb, int shaderType);
    void addUniformDeclarations(StringBuilder sb, int shaderType);
    void addFuncsDeclarations(StringBuilder sb, int shaderType);
    void addCalculations(StringBuilder sb, int shaderType);
}

File: android/app/src/main/java/com/darkraha/gldemos/gl/shader/HelperBuilder.kt
package com.darkraha.opengldemokt.gl.shader

interface HelperBuilder {
    fun addTypeDeclarations(sb: StringBuilder, shaderType: Int)
    fun addInputDeclarations(sb: StringBuilder, shaderType: Int)
    fun addExchangeDeclarations(sb: StringBuilder, shaderType: Int)
    fun addUniformDeclarations(sb: StringBuilder, shaderType: Int)
    fun addFuncsDeclarations(sb: StringBuilder, shaderType: Int)
    fun addCalculations(sb: StringBuilder, shaderType: Int)
}